.github/workflows/wheel_linux_ppc64le.yml

```diff
@@ -0,0 +1,64 @@
+name: Wheels
+on:
+  push:
+    branches:
+      - master
+
+env:
+  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+
+  # MacOS specific build settings
+  CIBW_BEFORE_ALL_MACOS: |
+    brew uninstall --force oclint
+    brew install gcc libomp
+
+  # Python build settings
+  CIBW_BEFORE_BUILD: |
+    pip install pybind11
+
+  # Skip PPC tests due to lack of numpy/scipy wheel support
+  CIBW_TEST_SKIP: "*-manylinux_{ppc64le}"
+
+  # Use Debian image for PPC
+  CIBW_MANYLINUX_PPC64LE_IMAGE: manylinux_2_24
```

Comment:
Does the little-endian vs big-endian subtlety matter? E.g., does it not make sense for PPC64 (i.e. without the LE)?

Reply:
x86 machines are little-endian on whole. Power CPUs tend to support both big and little endian on the same device, but keeping it as `le` makes porting between x86 and PPC much simpler. Most of the builder tooling for this generally assumes the little-endian Power ISA.